src/components/UserInfo.js

```diff
@@ -1,18 +1,27 @@
 export default class UserInfo {
-    constructor({ nameSelector, jobSelector }) {
-        this._nameElement = document.querySelector(nameSelector);
-        this._jobElement = document.querySelector(jobSelector);
+    constructor({ nameSelector, jobSelector, avatarSelector }) {
+      this._nameElement = document.querySelector(nameSelector);
+      this._jobElement = document.querySelector(jobSelector);
+      this._avatarElement = document.querySelector(avatarSelector);
     }
-
-    getUserInfo() {
-        return {
-            name: this._nameElement.textContent,
-            job: this._jobElement.textContent,
-        };
+  
+    getUserInformation() {
+      return {
+        name: this._nameElement.textContent,
+        job: this._jobElement.textContent,
+        avatar: this._avatarElement ? this._avatarElement : "",
+      };
     }
-
-    setUserInfo({ name, job }) {
+  
+    setUserInformation({ name, job, avatar }) {
         this._nameElement.textContent = name;
         this._jobElement.textContent = job;
+        if (avatar) {
+          this._avatarElement.src = avatar;
```

Comment:
`          this._avatarElement.src = avatar;
`

Please, use `setUserAvatar` here not to duplicate the code